src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -573,14 +584,27 @@ private IMemberReferenceExpression CreateBoundObjectInitializerMemberOperation(B
                     }
                     else
                     {
+                        var accessor = property.GetOwnOrInheritedSetMethod();
+                        if (accessor == null || boundObjectInitializerMember.ResultKind == LookupResultKind.OverloadResolutionFailure || accessor.OriginalDefinition is ErrorMethodSymbol)
+                        {
+                            Lazy<ImmutableArray<IOperation>> children = new Lazy<ImmutableArray<IOperation>>(
+                                 () =>
+                                 {
+                                     var builder = ArrayBuilder<IOperation>.GetInstance(boundObjectInitializerMember.Arguments.Length + 1);
+                                     builder.Add(CreateInstanceReferenceExpression());
```

Comment:
>builder.Add(CreateInstanceReferenceExpression()); [](start = 37, length = 49)

I don't think this node adds any value, it doesn't reflect anything present in syntax.

Reply:
I have excluded all compiler generated receiver when creating invalid expression.